src/routes/formsgSiteLaunch.ts

```diff
@@ -301,33 +301,39 @@ export class FormsgSiteLaunchRouter {
     formResponses: FormResponsesProps[],
     submissionId: string
   ) {
-    const launchResults = await Promise.all(
-      formResponses.map(this.launchSiteFromForm)
-    )
-    const successResults: DnsRecordsEmailProps[] = []
-    launchResults.forEach((launchResult) => {
-      if (launchResult.isOk()) {
-        successResults.push(launchResult.value)
-      }
-    })
+    try {
+      const launchResults = await Promise.all(
+        formResponses.map(this.launchSiteFromForm)
+      )
```

Comment:
hmm just beware that `Promise.all` means the resulting `Promise` succeeds only when **all** the promises succeed. 

if what you want is just for the promise to _complete_, the behaviour you want is `Promise.allSettled`

see [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/allSettled) for the `allSettled` behaviour

Reply:
good point, since this the 'launchSiteFromForm' never throws an error but returns it, i shall to keep this behaviour!